packages/apps/docs/src/docs/marmalade/what-is-marmalade.md

```diff
@@ -13,7 +13,7 @@ Marmalade is the name of the Kadena **token standard**.
 This token standard defines the interfaces for minting digital items with Kadena **smart contracts** and protecting those digital items with Kadena **token policies**.
 
 Digital items that are unique, one-of-a-kind assets are most commonly referred to as non-fungible tokens (NFTs).
-However, the Kadena token standard—influenced by the Ethereum [multi-token standard](https://ethereum.org/developers/docs/standards/tokens/erc-1155)—enables you to define, mint, and secure tokens that are fully fungible as interchangeable coins, partially fungible as a limited edition with a fixed supply, or completely non-fungible as unique items.
+However, the Kadena token standard is implemented as am multi-token standard that enables you to define, mint, and secure tokens that are fully fungible as interchangeable coins, partially fungible as a limited edition with a fixed supply, or completely non-fungible as unique items.
```

Comment:
I spotted a typo "a multi-token standard"

Reply:
I suggest we use the "poly-fungible standard" term instead of "multi-token standard" because we name our token interface as `poly-fungible-v3`